src/mainboard/protectli/vault_jsl/mainboard.c

```diff
@@ -44,6 +191,10 @@ void mainboard_silicon_init_params(FSP_S_CONFIG *params)
 
 	params->PcieRpDetectTimeoutMs[1] = 200;
 
+	/* On V1610 the NVMe disk sometimes do not appear in RP with defautl 0ms timeout*/
```

Comment:
```suggestion
	/* On V1610 the NVMe disk sometimes do not appear in RP with default 0ms timeout*/
```

Also, is this something specific to V1610? Maybe it should be done unconditionally, just in case.

Reply:
Saw this on V1610 only, because other boards have different PCIe topology and different link width.

Reply:
Fixed: https://github.com/Dasharo/coreboot/pull/536/commits/3d7ab16637462fcd0e04800593da543c0ba71297#diff-a84f53eaa895a8425691422e2a293ad75690452e20eb1fc750a56aade311ed0bR194